docs/weightonlyquant.md

```diff
@@ -169,7 +171,7 @@ inputs = tokenizer(prompt, return_tensors="pt").input_ids.to(device)
 qmodel = AutoModelForCausalLM.from_pretrained(model_name, load_in_4bit=True, device_map="xpu", trust_remote_code=True)
 
 # optimize the model with ipex, it will improve performance.
-qmodel = ipex.optimize_transformers(qmodel, inplace=True, dtype=torch.float16, quantization_config={}, device="xpu")
+qmodel = ipex.optimize_transformers(qmodel, inplace=True, dtype=torch.float16, woq=True, device="xpu")

```

Comment:
ipex.optimize_transformers is not support woq args anymore
and how about we hide it into generate api

Reply:
shall we use ipex.llm.xxx?

Reply:
@PenghuiCheng  Could you help to improve it if necessary?

Reply:
align with Penghui, the model after ipex.optimize_transformers doesn't support save, we plan to improve it later, not in this PR.